### PR TITLE
[DOCS-5670] Updates to ASM Getting Started Guide

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do?
Replaces https://github.com/DataDog/documentation/pull/18515

### Motivation
Updates to ASM getting started from @cyrbouchiat 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by buraizu** [comment]
Thanks, created DOCS-5689 to review

**Review by kayayarai — COMMENTED** [review]
review in progress :)

File: content/en/getting_started/application_security/_index.md [hunk 1]
```diff
@@ -24,72 +24,84 @@ further_reading:
...
+To benefit from all the ASM capabilities such as detections, automatic updates, and protection, first enable [Remote Configuration][15].
 
-4. Now that ASM is set up detecting suspicious requests on your service in real-time, **let the product run for a couple days**.
+### Enable ASM with in-app instructions
 
-If your services use recent versions of tracing libraries, ASM immediately [detects security vulnerabilities][13] in the upstream libraries that your services have as dependencies, and APM previews this information for you in its Security view. Optionally, explore ASM's consolidated [Vulnerabilities view][14] to see how it helps you triage and prioritize vulnerabilities.
+Go to the [ASM Setup page][1] and follow the instructions to get started. This includes:
+- Guided selection of services that would benefit from ASM.
+- Configuring your Datadog tracing libraries with an environment variable.
+- Restarting your services.
```
Comment (kayayarai) [inline content/en/getting_started/application_security/_index.md:59]: Move this section to after Enable RC? Or just delete?

File: content/en/getting_started/application_security/_index.md [hunk 2]
```diff
@@ -24,72 +24,84 @@ further_reading:
...
+  # Target existing service's routes
+  curl https://your-application-url/<EXISTING ROUTE> -A
+  'dd-test-scanner-log';
+  # Target non existing service's routes
+  curl https://your-application-url/<NON-EXISTING ROUTE> -A
+  'dd-test-scanner-log';
+  done{{< /code-block >}}
 
-3. **Archive the signal after it's been investigated.** If the signal is a false positive, set up a [passlist entry][9] to eliminate noisy signal patterns.
+3. A [Security Signal][6] should be generated within a few seconds.
```
Comment (kayayarai) [inline content/en/getting_started/application_security/_index.md:95]:
```suggestion
3. Go to [Security Signal][6] to see the signal that is generated after a few seconds.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/getting_started/application_security/_index.md [hunk 3]
```diff
@@ -24,72 +24,84 @@ further_reading:
...
 
-   - If the client IP is not well-resolved (for example, if it's showing an internal or proxy IP), [configure the client IP header][3].
+2. After you have identified a few services under security risks, proceed to [enable ASM][20].
 
-   - Open the **[Traces][8]** page and click one of the traces for a suspicious request. The Attack Flow diagram helps you understand all the services hit by the attack. Scroll down to the Security section to read about why this request was flagged as suspicious. Read more in [How Application Security Management Works][4].
+## Setup ASM
 
-   You don't need to examine every suspicious request individually. ASM generates _signals_ when your attention is required, which you can learn about in Phase 2.
+ASM relies on the same library as Datadog Application Performance Monitoring (APM). 
+To benefit from all the ASM capabilities such as detections, automatic updates, and protection, first enable [Remote Configuration][15].
```
[the remark this answers is not in the document]
Comment (cyrbouchiat) [reply content/en/getting_started/application_security/_index.md:49]: Agreed, would recommed to prioritize In-app instructions over Remote Config, as RC activation is full of frictions, so would be a secondary option

File: content/en/getting_started/application_security/_index.md (hunk 1 above)
Comment (cyrbouchiat) [reply content/en/getting_started/application_security/_index.md:56]: 1. In-APP instructions
2. Activation through RC

File: content/en/getting_started/application_security/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/getting_started/application_security/_index.md]: Great! I have moved the in-app instructions above RC

**Comment by aliciascott** [comment]
@cyrbouchiat approved via slack, merged

**Pull request merged**